tests/unit/project_api/test_project_api.py

```diff
@@ -381,3 +381,17 @@ def build_url(course_id, content_id, page_num=None):
         self.assertEqual(len(completions), len(all_responses))
         self.assertEqual([comp.id for comp in completions], [data['id'] for data in all_responses])
         self.assertEqual([comp.user_id for comp in completions], [data['user_id'] for data in all_responses])
+
+    def test_get_user_roles_for_course(self):
+        user_id = 1234
+        course_id = 4321
+        self.project_api.send_request = mock.Mock(return_value=[
+            {'role': 'foo', 'id': user_id},
+            {'role': 'bar', 'id': user_id},
+            {'role': 'baz', 'id': user_id},
+        ])
+        response = self.project_api.get_user_roles_for_course(user_id, course_id)
+        self.assertEqual(response, {'foo', 'bar', 'baz'})
```

Comment:
@jbzdak nit: please add a couple of ddt cases with different user_id and course_id. The reason is that if you follow TDD (and especially test-first approach), this test can be satisfied by a much simpler (and obviously wrong) implementation:

    def get_user_roles_for_course(self, user_id, course_id):
        self.send_request(GET, ('api/server/courses', course_id, 'roles'), query_params={'user_id': 1234})
        return {'foo', 'bar', 'baz'}

So, please add a couple of `ddt` cases with different course_id, user_id and roles assigned